src/backend/data/post.js

```diff
@@ -144,9 +145,13 @@ class Post {
       // link is the url to the post
       article.link,
       article.guid,
-      feed
+      feed,
+      article.author
     );
-    await Promise.all([post.save(), indexPost(post.text, post.id)]);
+    await Promise.all([
+      post.save(),
+      indexPost(post.text, post.id, post.title, post.updated, post.author),
```

Comment:
You can find `author` inside of `feed` (line 148). 
```suggestion
      indexPost(post.text, post.id, post.title, post.updated, post.feed.author),
```
Check [here](https://github.com/Seneca-CDOT/telescope/blob/3e0592038bca898e7df11e90611104d39e3b09ee/src/backend/data/feed.js#L27) what's inside of a `feed` object.

Reply:
I didn't know it~ I followed your suggestion and removed all my author parts.